simulation/g4simulation/g4bwd/autogen.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Note this file is expected to be executable `chmod 755` for many cases of compilation starting with running `autogen.sh` directly